### DISCUSSION TITLE
Return user to the same place in the Dashboard's listing of Aliases when an Alias is deleted

### DISCUSSION THREAD

**Discussion by WhamboMPS** [simple-login/app#579] [Feature Requests]
The [Dashboard](https://app.simplelogin.io/dashboard) lists all Alias that the user has created. It is sometimes useful to page through all Aliases page by page to see if anything is amiss, add a common set of Notes to Aliases, or whatever. However, if a user deletes an Alias the Dashboard presentation will return to the very first listing (using the most-recently used Alias default sort). Change this behavior so that the user is returned to the Alias listing in the same position where the now-deleted Alias was located (e.g. one Alias up or one Alias down from the now-deleted Alias). As a side note the soft and filtering criterion specified before the Alias was deleted should be retained (see issue #580).

**Comment by strawrhu** [comment]
YES! This has been a pet peeve of mine since first subscribing to Simple Login and accumulating enough aliases that they fill multiple pages.
